src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,838 @@
+ :search/cache-outdated "Cache is outdated. Please click the 'Re-index' button in the graph's dropdown menu."
+ :search-item/whiteboard "Whiteboard"
+ :search-item/page "Page"
+ :search-item/file "File"
+ :search-item/block "Block"
+ :search-item/no-result "No matched result"
+ :help/context-menu "Block context menu"
+ :help/markdown-syntax "Markdown syntax"
+ :help/org-mode-syntax "Org mode syntax"
+ :bold "Bold"
+ :italics "Italics"
+ :highlight "Highlight"
+ :strikethrough "Strikethrough"
+ :code "Code"
+ :untitled "Untitled"
+ :right-side-bar/help "Help"
+ :right-side-bar/switch-theme "Theme modes"
+ :right-side-bar/contents "Contents"
+ :right-side-bar/page-graph "Page graph"
+ :right-side-bar/history "(Dev) Undo/Redo history"
+ :right-side-bar/history-undos "Undos"
+ :right-side-bar/history-redos "Redos"
+ :right-side-bar/history-global "global"
+ :right-side-bar/history-pageonly "page only"
+ :right-side-bar/block-ref "Block references"
+ :right-side-bar/graph-view "Graph view"
+ :right-side-bar/all-pages "All pages"
+ :right-side-bar/whiteboards "Whiteboards"
+ :right-side-bar/flashcards "Flashcards"
+ :right-side-bar/new-page "New page"
+ :right-side-bar/show-journals "Show Journals"
+ :right-side-bar/separator "Right sidebar resize handler"
+ :right-side-bar/toggle-right-sidebar "Toggle right sidebar"
+ :right-side-bar/pane-close "Close"
+ :right-side-bar/pane-close-others "Close others"
+ :right-side-bar/pane-close-all "Close all"
+ :right-side-bar/pane-collapse "Collapse"
+ :right-side-bar/pane-collapse-others "Collapse others"
+ :right-side-bar/pane-collapse-all "Collapse all"
+ :right-side-bar/pane-expand "Expand"
+ :right-side-bar/pane-expand-all "Expand all"
+ :right-side-bar/pane-open-as-page "Open as page"
+ :right-side-bar/pane-more "More"
+ :left-side-bar/switch "Switch to:"
+ :left-side-bar/journals "Journals"
+  :left-side-bar/create "Criar"
+ :left-side-bar/new-page "Nova página"
+ :left-side-bar/new-whiteboard "Novo whiteboard"
+ :left-side-bar/nav-favorites "Favoritos"
+ :left-side-bar/nav-recent-pages "Recentes"
+ :page/something-went-wrong "Algo deu errado"
+ :page/logseq-is-having-a-problem "Logseq está com um problema. Para tentar fazê-lo voltar a funcionar, siga as etapas seguras a seguir:"
+ :page/step "Etapa {1}"
+ :page/try "Tentar"
+ :page/slide-view "Visualizar como slides"
+ :page/slide-view-tip-go-fullscreen (fn [] [[:span.opacity-70 "Dica: pressione "] [:code "f"] [:span.opacity-70 " para ir para tela cheia"]])
+ :page/delete-confirmation "Tem certeza de que deseja excluir esta página e seu arquivo?"
+ :page/open-in-finder "Abrir no diretório"
+ :page/open-with-default-app "Abrir com o aplicativo padrão"
+ :page/make-public "Torná-lo público para publicação"
  :page/version-history "Ver histórico da página"
- :page/make-private "Tornar privada"
+ :page/open-backup-directory "Abrir diretório de backups de página"
+ :page/make-private "Torná-lo privado"
```

Comment:
Here we are referring to a page, so we can use "Torná-la" or leave as "Tornar".

Reply:
Done